api/pyproject.toml

```diff
@@ -58,7 +58,7 @@ rope = "^1"
 jsonpickle = "^3.0.0"
 pytest-watch = "^4.2.0"
 pytest-testmon = "^2.0.0"
-ruff = "^0.0.286"
+ruff = "^0.0.287"
```

Comment:
This is weird, main is already using ruff @ 0.0.287

Reply:
Likely due to a merge from main instead of rebasing. Same for poetry.lock and package.json.